packages/pluggableWidgets/datagrid-date-filter-web/typings/DatagridDateFilterProps.d.ts

```diff
@@ -29,5 +30,6 @@ export interface DatagridDateFilterPreviewProps {
     placeholder: string;
     adjustable: boolean;
     screenReaderButtonCaption: string;
+    screenReaderCalendarCaption: string;
```

Comment:
You don't use this property inside the editor preview. It helps a11y on the Studio side for developers.

Reply:
Fixed for all filters